app/views/project/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :pre_content_nav do %>
+<%= link_to "Back", root_path, class: "govuk-back-link" %>
+<% end %>
+
+<%= form_with model: @project, url: project_index_path do |form| %>
+
+  <%= form.govuk_error_summary %>
+
+  <h1 class="govuk-heading-l"><%= t("project.new.title") %></h1>
+
+  <%= form.govuk_number_field :urn,
```

Comment:
This is super interesting! I booted up the app and could never see the 'not a number' error!? After much head scratching I tracked it down to this.

Because we are using `govuk_number_field` modern browsers do not submit anything else, so providing 'three' actually submits '' (empty string).

The feature spec you have passes becuase the headless browser Capybara uses is not modern enough to handle this nuance so it submits 'three'.

Because we want to help users I think it is better to change this to `govuk_text_field` so we can display helpful errors, I also think the behaviour is more consistent - what do you think?